### ISSUE TITLE
How to insert a subtitle link?

### ISSUE THREAD

**Issue by Vampireos** [Arnavion/libjass#82]
hi, prompt and have the opportunity to insert into the subtitles link?

**Comment by Arnavion** [comment]
I don't understand what you mean.

**Comment by XCanG** [comment]
May be he mean something like html code:
```HTML
<a href="http://site.com/url-what-he-need/">Here is a link text, may be different, than URL</a>
```
into subtitles, like making some ads, while user will watch the video.

**Comment by Vampireos** [comment]
Yes, I'm talking about links like `<a href="http://site.com/..">Link name</a>`

**Comment by Arnavion** [comment]
No, you cannot get libjass to render HTML in subtitles as HTML.
